CONTRIBUTING.md

```diff
@@ -19,14 +19,10 @@ it gives the developers room to grow!
 
 To add a repository to the curated list, please
 
-* Locate the [Issue][_issu] in which the repository was nominated,
```

Comment:
My initial reaction was that it's *OK* if there's an issue, and if so, we should reference it, but the more I think about it, for this use case, I think if somebody thinks highly of a project, they should just do a pull request and not (effectively) suggest that somebody else should do a pull request.

tl;dr: I approve of removing all discussion of issues

Reply:
Yeah, in thinking about your question, that was my conclusion, too.

Merging. @agmorey and @mnewrock, if you have objections, please file issues.